### PR TITLE
[Example] Disable ENABLE_DEV_MODE definition to use production on sim.

### PR DESCRIPTION
**Pull request by alloy**
Self merging!

### CODE REVIEW

File: Pod/Assets/Emission.js.meta [hunk 1]
```diff
@@ -0,0 +1 @@
+%���>�'�A��%:1���>
```
Comment (orta) [inline Pod/Assets/Emission.js.meta]: o.0

Comment (alloy) [inline Pod/Assets/Emission.js.meta]: Yeah… I have no clue, thanks for the reminder.

Comment (orta) [inline Pod/Assets/Emission.js.meta]: I saw this when I was doing my work on deployment too, might be a new thing in RN

Comment (alloy) [inline Pod/Assets/Emission.js.meta]: https://github.com/facebook/react-native/commit/ae6afcf3f65a84fa45dd999a916e33285b41d640

Comment (orta) [inline Pod/Assets/Emission.js.meta]: 👍

Comment (alloy) [inline Pod/Assets/Emission.js.meta]: https://github.com/facebook/react-native/issues/8686#issuecomment-244126580

**Pull request merged**